src/App.js

```diff
@@ -6,29 +6,31 @@ import MovieList from './components/MovieList'
 import SearchBar from './components/SearchBar'
 
 const DEFAULT_STATUS = 'Search a massive database of movies and TV shows!'
-const SearchStatus = ({text}) => <h1>{text}</h1>
+const SearchStatus = ({text, color}) => <h1 className={color}>{text}</h1>
 
 class App extends Component {
   constructor() {
     super()
     this.state = {
       movies: [],
       search: '',
-      status: DEFAULT_STATUS
+      status: DEFAULT_STATUS,
+      color: 'black'
     }
     this.handleSearchChange = this.handleSearchChange.bind(this)
     this.handleSubmit = this.handleSubmit.bind(this)
     this.clearSearch = this.clearSearch.bind(this)
   }
 
   fetchMovies(search){
-    const { movies, status } = this.state
+    const { movies, status, color } = this.state
     let typedSearch = `http://www.omdbapi.com/?apikey=df1379a2&s=${search}`
     axios.get(typedSearch)
     .then(({ data }) => {
       const status = data.Error || ''
       const movies = data.Search || []
       this.setState({ status, movies })
+      data.Error && this.setState({ color: 'red' })
```

Comment:
The goal here is to only call `setState` once. I would do:

```
const color = data.Error ? 'red' : 'black'
this.setState({ status, movies, color })
```

Reply:
much nicer than mine. thanks! done.